server/routes/imageupload/imageupload-router.js

```diff
@@ -0,0 +1,66 @@
+import morgan from('morgan');
+import express from ('express');
+import multer from ('multer');
+import path from ('path');
+import knex from ('knex');
+
+const db = knex(
+{
+  client: 'pg',
+  connection: {
+    host: '',
+    user: 'trees',
+    password: 'trees3r4t',
+    database: 'treedb',
+  },
+}
+);
+const imageUpload = multer({
+dest: 'images',
+});
+
+
+const imageuploadRouter = express.Router();
+
+
+imageuploadRouter.use(express.json());
+imageuploadRouter.use(morgan('dev'));
+
+
+imageuploadRouter.post('/image', imageUpload.single('image'), (req, res) => {
+const { filename, mimetype, size } = req.file;
+const filepath = req.file.path;
+db
+ .insert({
+  filename,
+  filepath,
+  mimetype,
+  size,
+ })
+ .into('DATABASE TABLE NAME')
+ .then(() => res.json({ success: true, filename }))
+ .catch(err => res.json({ success: false, message: 'upload failed', stack: err.stack }));
+});
+
+
+imageuploadRouter.get('/image/:filename', (req, res) => {
+const loaded = false;
+const { filename } = req.params;
+if(loaded === false){
```

Comment:
@Spicytuna24 is this `if` statement supposed to be here? I see you're checking if `loaded` is false when it was already set as such two lines above.

Reply:
That'd old and I need to remove that